selection_tools/include/Particle.hpp

```diff
@@ -29,8 +29,12 @@ class Particle
   double eta();
   double momentum() const;
   double pt() const;
-  
-  
+  double Px() const;
+  double M() const;
+  double E() const;
+  TVector3 boost() const;  
```

Comment:
I guess this should be called "get_boost", since it doesn't boost the particle, but rather returns its boost (right?)